docs/architecture/deep-dives/passkeys/index.md

```diff
@@ -0,0 +1,4 @@
+# FIDO2 and Passkeys
+
+This is not a comprehensive guide to FIDO2, but a summary of the standard's crucial aspects for
+working within the Bitwarden context.
```

Comment:
🎨 Instead of this being a mostly-empty page, place all the content on the "overview" page here. I think this flows better and seems to be what other deep dives do, plus it's less time spent navigating.

A couple of the architecture pages have a sub-page with an overview, but I think that's actually an area for improvement.